tests/devices/test_default_qubit.py

```diff
@@ -2296,43 +2296,28 @@ def test_s_inverse():
         expected = np.array([1.0, -1.0j]) / np.sqrt(2)
         assert np.allclose(dev.state, expected, atol=tol, rtol=0)
 
-    def test_inverse_S_decomposition(self, tol, mocker, monkeypatch):
-        """Test that applying the inverse of the S gate
-        works when the inverse S gate is decomposed"""
+    def test_inverse_decomposition(self, tol, mocker):
+        """Test that the expansion is inverted when inverse gate is applied and the gate isn't supported."""
         dev = qml.device("default.qubit", wires=1)
         spy = mocker.spy(dev, "execute")
 
-        patched_operations = dev.operations.copy()
-        patched_operations.remove("S")
```

Comment:
Patching the list of operations now longer affects what operations are supported.